Makefile

```diff
@@ -1,6 +1,16 @@
 SHELL:=/bin/bash
 L0_VERSION?=$(shell git describe --tags)
 
+build: 
+	$(MAKE) -C api build        
+	$(MAKE) -C setup build       
+	$(MAKE) -C cli build      
+
+install:
+	$(MAKE) -C cli install
+	$(MAKE) -C setup install
+	$(MAKE) -C api release
```

Comment:
Is the `install` target ever used? Since we aren't really doing CI at the moment, I can't think of a use for it. Also, I'm not sure it makes sense to have `api release` under this target?

Reply:
well, we already have a `build` target - I don't think there is anything to install w the api.

Reply:
Maybe make a target to run it on docker then. I just want a command to setup everything for a dev.

Reply:
Either way I can remove this. Should I merge the develop stuff here so we can close out this issues.